engine/systemd/unit.go

```diff
@@ -221,18 +216,11 @@ func (b *unitBuilder) buffer() (*bytes.Buffer, error) {
 	return bytes.NewBufferString(unit), b.err
 }
 
-func (b *unitBuilder) convertToSystemdRestartPolicy(restart string) (policy string, err error) {
-	switch {
-	case restart == "no":
-		policy = "no"
-	case restart == "always" || restart == "":
-		policy = "always"
-	case strings.HasPrefix(restart, "on-failure"):
-		policy = "on-failure"
-	default:
-		err = fmt.Errorf("restart policy not supported: %s", restart)
+func (b *unitBuilder) convertToSystemdRestartPolicy(restart int) (policy string) {
+	if restart > 0 {
+		return "always"
 	}
-	return
+	return "no"
```

Comment:
Due to `No description provided.`, so is infinite restarting still supported? How to set it if so?

Reply:
Or limited retrying is not supported in `systemd`?